packages/tas_keybind/init.lua

```diff
@@ -0,0 +1,72 @@
+tas = tas or {}
+local oldPlayerControls = {}
```

Comment:
I don't think we're doing any cleanup here for when players leave the game and rejoin?